babel.config.js

```diff
@@ -14,6 +14,7 @@ module.exports = api => {
         plugins: [
             '@babel/plugin-proposal-class-properties',
             '@babel/plugin-proposal-object-rest-spread',
+            '@babel/plugin-proposal-private-methods',
```

Comment:
Should this move to the overrides as well?

Reply:
That's a good idea, I'll update it!